versions/3.0.4.md

```diff
@@ -208,62 +245,62 @@ These limitations are expected to be addressed in a future release.
-`integer` | `int64` | signed 64 bits (a.k.a long)
-`number` | `float` | |
-`number` | `double` | |
-`string` | `byte` | base64 encoded characters - [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4)
-`string` | `binary` | any sequence of octets
-`string` | `date` | As defined by `full-date` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6)
-`string` | `date-time` | As defined by `date-time` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6)
-`string` | `password` | A hint to obscure the value.
+| [`type`](#data-types) | [`format`](#data-type-format) | Comments                                                                                  |
+| -------------------- | --------------------------- | ----------------------------------------------------------------------------------------- |
+| `integer`            | `int32`                     | signed 32 bits                                                                            |
+| `integer`            | `int64`                     | signed 64 bits (a.k.a long)                                                               |
+| `number`             | `float`                     |                                                                                           |
+| `number`             | `double`                    |                                                                                           |
+| `string`             | `byte`                      | base64 encoded characters - [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4)   |
+| `string`             | `binary`                    | any sequence of octets                                                                    |
+| `string`             | `date`                      | As defined by `full-date` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6) |
+| `string`             | `date-time`                 | As defined by `date-time` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6) |
+| `string`             | `password`                  | A hint to obscure the value.                                                              |
 
-#### <a name="binaryData"></a>Working With Binary Data
+#### Working With Binary Data
 
 Two formats, `binary` and `byte`, describe different ways to work with binary data:
 
-* `binary` is used where unencoded binary data is allowed, such as when sending a binary payload as an HTTP message body, or as part of a `multipart/*` payload that allows binary parts
-* `byte` is used where binary data is embedded in a text-only format such as `application/json` or `application/x-www-form-urlencoded`
+- `binary` is used where unencoded binary data is allowed, such as when sending a binary payload as an HTTP message body, or as part of a `multipart/*` payload that allows binary parts
+- `byte` is used where binary data is embedded in a text-only format such as `application/json` or `application/x-www-form-urlencoded`
```

Comment:
I'm a bit puzzled by this list indicator change.  I feel like we have always consistently used `*` for this and the change introduces a lot of noise.  It's not a hill I want to die on, but I'd prefer `*` (both because it's what I'm used to scaning visually for when I edit, and to trim down this diff)

Reply:
`-` has evolved as a standard in markdown tools, the linter actually checks that the lists are consistent but I can totally enforce `*` instead if that's our normal.

Reply:
>  `-` has evolved as a standard in markdown tools

UGH why????  I'm a bit reluctant to start introducing digressions.  I do think `*` makes lists much easier to read in the source, but maybe that's because I'm more used to it.  It's just... bullet points... `*` looks like a bullet point, `-` does not!